src/openttd.cpp

```diff
@@ -386,9 +386,6 @@ void MakeNewgameSettingsLive()
 		_settings_game.ai_config[c] = nullptr;
 		if (_settings_newgame.ai_config[c] != nullptr) {
 			_settings_game.ai_config[c] = new AIConfig(_settings_newgame.ai_config[c]);
-			if (!AIConfig::GetConfig(c, AIConfig::SSS_FORCE_GAME)->HasScript()) {
-				AIConfig::GetConfig(c, AIConfig::SSS_FORCE_GAME)->Change(nullptr);
-			}
```

Comment:
This change feels wrong

Reply:
This was the fix for #6460. The "start_date" parameter for random ais was at times not part of their configuration. This PR undoes that fix because "start_date" is no longer part of an AI configuration.